src/pages/Productdetail/ImageSlider.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import './ImageSlider.scss';
+
+class ImageSlider extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      sliderIndex: 0,
+      productSlideImg: [],
+    };
+  }
+
+  componentDidMount() {
+    fetch('http://localhost:3000/data/productDetail.json', {})
+      .then(res => res.json())
+      .then(data => {
+        this.setState({
+          productSlideImg: data,
+        });
+      });
+  }
+
+  onChangeDetailImg = index => {
+    if (this.state.productSlideImg.length <= index) index = 0;
+    if (index < 0) index = this.state.productSlideImg.length - 1;
+
+    this.setState({ sliderIndex: index });
+  };
+
+  render() {
+    const { productSlideImg } = this.state;
+    return (
+      <div className="productDetail">
+        <div className="ProductDetailImgTable">
+          <div className="productDetailImg">
+            <div className="mainSlide">
+              <div
+                className="mainSlideImgs"
+                style={{
+                  transform: `translate3d(${
+                    -500 * this.state.sliderIndex
+                  }px,0px,0px)`,
+                }}
+              >
+                {productSlideImg?.map((element, index) => {
```

Comment:
optional chaining 잘 적용해주셨네요 👍

Reply:
element라는 통상적인이름보다 매개변수명을 의미가 드러나게 적어줘야 가독성 측면에서 좋습니다. 
```suggestion
                {productImgList?.map((productImg, index) => {
```
위와 같은 식으로요, 위는 하나의 예시이고 고민해보셔서 적합한 이름으로 선정해주시면 됩니다!

Reply:
네 이미지로 변경했씁니다! 감사합니다!